kodi_addon_checker/check_addon.py

```diff
@@ -124,3 +127,15 @@ def _get_addons(xml_url: str):
         LOGGER.error(errrt)
     except requests.exceptions.ConnectTimeout as errct:
         LOGGER.error(errct)
+
+
+def check_schema(report: Report, Addon_xml, schema_path):
+    for xsd in os.listdir(schema_path):
+        schema = xmlschema.XMLSchema(schema_path + "/" + xsd)
+        try:
+            if schema.validate(Addon_xml):
```

Comment:
validate doesn't return anything, it only raises exceptions

Reply:
I have added validate just for now. I'll replace it with `schema.is_valid` after everything works

Reply:
But then you can't tell what's wrong.